filament/include/filament/View.h

```diff
@@ -637,6 +637,33 @@ class UTILS_PUBLIC View : public FilamentAPI {
      */
     bool isFrontFaceWindingInverted() const noexcept;
 
+    /**
+     * Enables use of the stencil buffer. This API is currently a WIP and experimental.
+     *
+     * The stencil buffer is an 8-bit, per-fragment unsigned integer stored alongside the depth
+     * buffer. The stencil buffer is cleared at the beginning of a frame and discarded after the
+     * color pass.
+     *
+     * Each fragment's stencil value is set during rasterization by specifying stencil operations on
+     * a Material. The stencil buffer can be used as a mask for later rendering by setting a
+     * Material's stencil comparison function and reference value. Fragments that don't pass the
+     * stencil test are then discarded.
+     *
+     * Post-processing must be enabled in order to use the stencil buffer.
```

Comment:
why is that a requirement?

Reply:
At the moment, the default render target does not allocate a Stencil buffer. Maybe we should.
Technically the requirement is either post-processing OR MSAA must be enabled.

Reply:
Ah right. Maybe we should add a flag to the swapchain creation to specify it needs to support stencil? And say that it's a requirement if post-process is not used?

Reply:
Yeah that's a good idea